packages/core/src/utils/public.ts

```diff
@@ -290,3 +290,15 @@ export function getOsInfo() {
     });
   });
 }
+
+/**
+ * shuffle array
+ * @param array
+ */
+export function shuffle(array: any[]) {
```

Comment:
Need a return value and unit tests for this function :)

Reply:
shuffle will just operate original array (just like python's built-in shuffle function). I have added unit tests